packages/ui/src/components/Question/QuestionsAccordion.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Accordion } from '@chakra-ui/react';
+
+import { Question, QuestionProps } from './Question';
+
+export type QuestionsAccordionProps = {
+  items: (QuestionProps & { id: number | string })[];
```

Comment:
💭 The `id` prop seems unnecessary here. If we need it only to be the `key` we can use the question's title instead.